src/transaction/boot_sr.c

```diff
@@ -3189,6 +3189,12 @@ xboot_shutdown_server (REFPTR (THREAD_ENTRY, thread_p), ER_FINAL_CODE is_er_fina
     }
   else if (get_server_type () == SERVER_TYPE_TRANSACTION)
     {
+      // NOTE: passive transaction server, regarding replication: since
```

Comment:
The note is right. But I don't think it is justified to add comments for all the things we don't have to do. Please remove it.

Reply:
Removed assert from `passive_transaction_server` dtor. Replaced with destroying `m_replicator` and moved the comment there.